### PR TITLE
Added feauture, when input is given. Cards is generated

### LINKED ISSUE #37: When input is given in header make a request to the backend to fetch all repos of the provided user
## Solution proposal

- [ ] On input in input field (header) a request should be sent to the backend

- [ ] The response should be passed on to ´´´ generateRepoCard``` and then that response should be rendered to the wrapper.

## Definition of done

- [ ] A list of repos are rendered when input is given to the header.

### PR DESCRIPTION
**Pull request by te4-andre-skvarc**
## What issue did you implement or fix?
Fixed #37 

## Summary
Added so when a GitHub username is given, wrapper generates cards.
Also added, invalid request. Such as non existing username or that
the server is not responding. 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: public/js/main.js [hunk 1]
```diff
@@ -1,10 +1,30 @@
+document.querySelector('#for').addEventListener('submit', async (e) => {
```
Comment (te4-david-jensen) [inline public/js/main.js:1]: replace `document.querySelector('#for')` with `QS(document, '#for')`

**Review by te4-david-jensen — CHANGES_REQUESTED** [review]
looks go over all, just check comment

File: public/js/main.js (hunk 1 above)
Comment (te4-andre-skvarc) [reply public/js/main.js:1]: Okey, I will use this in the future!

File: public/js/main.js [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
```
Comment (te4-david-jensen) [inline public/js/main.js:1]:
```suggestion
QS(document, '#for').addEventListener('submit', async (e) => {
```

**Review by te4-david-jensen — APPROVED** [review]
👍  :shipit:

**Review by te4-adrian-almetun-smeds — APPROVED** [review]
After @te4-david-jensen suggestion I approve of this. 👍  🐔

**Pull request merged**